src/sentry/integrations/jira/utils/choice.py

```diff
@@ -1,13 +1,11 @@
-def build_user_choice(user_response, user_id_field):
+def build_user_choice(user_response):
```

Comment:
Looking at this change, I'm not sure it's entirely necessary to decouple Jira and Jira Server (maybe, idk).

It makes sense since this `"accountId"` string isn't likely to change, you can hard code it here, but I don't love that if we had to change it, we'd make the change in two places.

It's fine if we want to modify this `build_user_choice` function, but I still think it should use `client.get_user_id_field()`  in some way (maybe directly, instead of as a param), so they point to the same instance of the string "accountId".

Reply:
I put back `client.user_id_field()` but kept them separate since it feels weird to share a single file and copy/paste everything else even though they're identical.